src/ui/full-ui.component.js

```diff
@@ -45,6 +45,7 @@ export default class FullUI extends Component {
     return (
       <div>
         <button
+          part={`button${this.atLeastOneOverride() ? " with-override" : ""}`}
```

Comment:
To match the documentation, should be `with-overrides`. Plural makes more sense here since there are often multiple overrides.

```suggestion
          part={`button${this.atLeastOneOverride() ? " with-overrides" : ""}`}
```